index.js

```diff
@@ -1,20 +1,23 @@
 const bookmarks = document.querySelectorAll('[data-js="bookmark"]');
+const showAnsweButtons = document.querySelectorAll(
```

Comment:
Maybe you want to check on that typo before you wonder why "showAnswerButtons" does not work.